release-content/0.14/release-notes/12792_Implement_Auto_Exposure_plugin.md

```diff
@@ -1,4 +1,13 @@
-<!-- Implement Auto Exposure plugin -->
-<!-- https://github.com/bevyengine/bevy/pull/12792 -->
+Since **Bevy 0.13**, you can configure the the EV100 of a camera, which allows you to adjust the exposure of the camera in a physically based way. This also allows you to dynamically change the exposure values for various effects. However, this is a manual process and requires you to adjust the exposure values yourself.
 
-<!-- TODO -->
+**Bevy 0.14** introduces **Auto Exposure**, which automatically adjusts the exposure of your scene based on the brightness of the scene. This can be useful when you want to create the feeling of a very high dynamic range, since your eyes also adjust to large changes in brightness. Note that this is not a replacement for hand-tuning the exposure values, rather an additional tool that you can use to create dramatic effects when brightness changes rapidly.
+
+<video controls><source src="auto_exposure.mp4" type="video/mp4"/></video>
+
+Bevy's Auto Exposure is implemented by making a **histogram** of the scene's brightness in a post processing step. The exposure is then adjusted based on the average of the histogram. Because the histogram is calculated using a compute shader, Auto Exposure is **not available on WebGL**. It's also not enabled by default, so you need to add the [`AutoExposurePlugin`](https://docs.rs/bevy/0.14/bevy/core_pipeline/auto_exposure/struct.AutoExposurePlugin.html) to your app.
```

Comment:
As an aside, I'm curious why it's a non-default plugin. Usually, we tend to make all plugins part of PbrPlugin and enabled by default, and simply early-exit if there's no corresponding camera setting. The only exception is stuff like TAA which disables MSAA when you add the plugin.

Reply:
That's a really good question, I did this because it wouldn't work on the webgl backend, but it could of course be turned off by feature gates instead. Would it be too late to fix this for 0.14?

Reply:
Nope, submit a PR to change this.

Reply:
Doesn't need feature gating either. Just note in the docs that it's not WebGL2 compatible, and it's up to users to not use it on those platforms. Feature gating makes it harder to find, and there's already a ton of features.

Reply:
Alright, I don't have a lot of time at the moment, I will hopefully do this around the weekend

Reply:
Resolved when this PR gets merged: https://github.com/bevyengine/bevy/pull/13963